string user
---
int32 result
